global_frame: map
robot_base_frame: base_link
update_frequency: 5.0
publish_frequency: 5.0
map_update_rate: 5.0

#set if you want the voxel map published
# publish_voxel_map: false
##set to true if you want to initialize the costmap from a static map
static_map: true
#
##begin - COMMENT these lines if you set static_map to true
#rolling_window: true
#width: 4.0
#height: 4.0
#resolution: 0.05
# always_send_full_costmap: false
##end - COMMENT these lines if you set static_map to true
#
##START VOXEL STUFF
#map_type: voxel
#origin_z: 0.0
#z_resolution: 0.2
#z_voxels: 10
#unknown_threshold: 10
#mark_threshold: 0
##END VOXEL STUFF
#
transform_tolerance: 0.3
obstacle_range: 4.0
#max_obstacle_height: 2.0
raytrace_range: 4.0
# footprint: [0.0, 0.0]
robot_radius: 0.01
#footprint_padding: 0.01
inflation_radius: 0.20
cost_scaling_factor: 0.1
lethal_cost_threshold: 100
observation_sources: laser_scan_sensor
laser_scan_sensor: {sensor: laser, data_type: LaserScan, expected_update_rate: 0.1, topic: scan,
  observation_persistence: 0.0, marking: true, clearing: true, max_obstacle_height: 0.4, min_obstacle_height: 0.08}
